hrm

```
2018-11-15 21:29:58.875327 | static | api-lint runtests: commands[0] | flake8
2018-11-15 21:29:58.875391 | static | setting PATH=/home/zuul/src/github.com/ansible/awx/.tox/api-lint/bin:/usr/local/bin:/usr/bin
2018-11-15 21:29:58.878098 | static |   /home/zuul/src/github.com/ansible/awx$ /home/zuul/src/github.com/ansible/awx/.tox/api-lint/bin/flake8
2018-11-15 21:30:11.147364 | static | ./awx/main/tests/docs/test_swagger_generation.py:175:15: W605 invalid escape sequence '\s'
2018-11-15 21:30:11.147528 | static | ./awx/main/tests/docs/test_swagger_generation.py:176:42: W605 invalid escape sequence '\,'
```

--- a/Makefile
+++ b/Makefile
@@ -578,7 +578,8 @@ docker-compose-genschema:
 docker-compose-validate-schema:
 	$(MAKE) docker-compose-genschema
 	curl https://s3.amazonaws.com/awx-public-ci-files/schema.json -o reference-schema.json
-	diff -u schema.json reference-schema.json
+	# Ignore differences in whitespace with -b
+	diff -u -b schema.json reference-schema.json
 
 docker-compose-clean:
 	cd tools && CURRENT_UID=$(shell id -u) TAG=$(COMPOSE_TAG) DEV_DOCKER_TAG_BASE=$(DEV_DOCKER_TAG_BASE) docker-compose run --rm -w /awx_devel --service-ports awx make clean

--- a/awx/main/tests/docs/test_swagger_generation.py
+++ b/awx/main/tests/docs/test_swagger_generation.py
@@ -122,7 +122,7 @@ class TestSwaggerGeneration():
             pattern = pattern.replace('{id}', '[0-9]+')
             pattern = pattern.replace(r'{category_slug}', r'[a-zA-Z0-9\-]+')
             for path, result in swagger_autogen.items():
-                if re.match('^{}$'.format(pattern), path):
+                if re.match(r'^{}$'.format(pattern), path):
                     for key, value in result.items():
                         method, status_code = key
                         content_type, resp, request_data = value
@@ -168,13 +168,13 @@ class TestSwaggerGeneration():
             # replace ISO dates w/ the same value so we don't generate
             # needless diffs
             data = re.sub(
-                '[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}.[0-9]+Z',
-                '2018-02-01T08:00:00.000000Z',
+                r'[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}.[0-9]+Z',
+                r'2018-02-01T08:00:00.000000Z',
                 data
             )
             data = re.sub(
-                '''(\s+"client_id": ")([a-zA-Z0-9]{40})("\,\s*)''',
-                '\\1xxxx\\3',
+                r'''(\s+"client_id": ")([a-zA-Z0-9]{40})("\,\s*)''',
+                r'\1xxxx\3',
                 data
             )
             f.write(data)

